__init__.py

```diff
@@ -0,0 +1 @@
+#does this file need to be here?
```

Comment:
This file is needed to help python to find other .py file

Reply:
Yep leftvoer from my "review"